docs/source/resources/software_stack.rst

```diff
@@ -0,0 +1,238 @@
+.. role:: raw-html(raw)
+    :format: html
+
+We offer the following software stacks:
+
+* `Merlin Inference <table-1>`_: Allows you to deploy NVTabular workflows and HugeCTR or TensorFlow models to the Triton Inference server for production.
+* `Merlin Training <table-2>`_: Allows you to do preprocessing and feature engineering with NVTabular so that you can train a deep learning recommendation model with HugeCTR.
+* `Merlin TensorFlow Training <table-3>`_: Allows you to do preprocessing and feature engineering with NVTabular so that you can train a deep learning recommendation model with TensorFlow.
+* `Merlin PyTorch Training <table-4>`_: Allows you to do preprocessing and feature engineering with NVTabular so that you can train a deep learning recommendation model with PyTorch.
+
+The following tables provide the software and model versions that NVTabular version 0.6 supports.
+
+:raw-html:`<br/>`
+
+.. _table-1:
+:raw-html:`<p align="center"><b>Table 1: Software stack matrix for the Merlin Inference (merlin-inference) image</b></p>`
```

Comment:
The docs build is complaining about this line: 
```
/var/jenkins_home/workspace/nvtabular_tests/nvtabular/docs/source/resources/software_stack.rst:16:Explicit markup ends without a blank line; unexpected unindent.
```

I'm not sure why though since afaict the next line is blank?

Reply:
@benfred - Can you try removing the blank line and see if it takes it?

Reply:
I made a couple fixes here https://github.com/NVIDIA/NVTabular/pull/934/commits/a873b16eac0f79e06fa40b4b1869dbbbcdbb33fa that should resolve the build issues. Mainly adding some blank lines, adding a title, and fixing some internal urls .

Reply:
Good catch on the title and URLs ...this is what happens when you format in MD, convert to HTML, and then end up reformatting in RST (LOL)